hypergan/samplers/random_walk_sampler.py

```diff
@@ -9,32 +9,38 @@ def __init__(self, gan, samples_per_row=8):
         self.y = None
         self.x = None
         self.step = 0
-        self.steps = 30
+        self.steps = 8
         self.target = None
 
     def _sample(self):
         gan = self.gan
-        z_t = gan.encoder.sample
```

Comment:
Yep I'm going to revert this file for now.  RandomWalk looks bad on AlphaGAN without switching encoders/samples.  Maybe we need to refactor samplers to work better on different architectures.